core/templates/pages/topics-and-skills-dashboard-page/topics-list/topics-list.component.html

```diff
@@ -84,117 +85,117 @@
+              <span *ngIf="topic.classroom">{{topic.classroom}}</span>
+              <span *ngIf="!topic.classroom">Unassigned</span>
             </div>
           </div>
           <div class="topic-item-options">
-            <p class="fas fa-ellipsis-v topic-edit-box-icon" ng-click="$ctrl.changeEditOptions(topic.id)"></p>
-            <div ng-if="$ctrl.showEditOptions(topic.id)" class="topic-edit-options">
-              <a class="topic-edit-options-edit" ng-href="<[$ctrl.getTopicEditorUrl(topic.id)]>">
+            <p class="fas fa-ellipsis-v topic-edit-box-icon" (click)="changeEditOptions(topic.id)"></p>
+            <div *ngIf="showEditOptions(topic.id)" class="topic-edit-options">
+              <a class="topic-edit-options-edit" [href]="getTopicEditorUrl(topic.id)">
                 <i class="fa fa-pen"></i>
                 <span>Edit</span>
               </a>
-              <a ng-if="canDeleteTopic()" ng-click="$ctrl.deleteTopic(topic.id, topic.name)">
+              <a *ngIf="userCanDeleteTopic" (click)="deleteTopic(topic.id, topic.name)">
                 <i class="fas fa-trash-alt"></i>
                 <span>Delete</span>
               </a>
             </div>
           </div>
         </div>
         <div class="topic-item-description">
-          <span><[topic.description]></span>
+          <span>{{topic.description}}</span>
         </div>
         <div class="topic-item-entities">
           <div class="topic-item-entity">
             <div class="topic-item-value">
-              <span><[topic.canonicalStoryCount]></span>
+              <span>{{topic.canonicalStoryCount}}</span>
             </div>
             <div class="topic-item-type">
               Published Stories
             </div>
           </div>
           <div class="topic-item-entity">
             <div class="topic-item-value">
-              <span><[topic.subtopicCount]></span>
+              <span>{{topic.subtopicCount}}</span>
             </div>
             <div class="topic-item-type">
               Subtopics
             </div>
           </div>
           <div class="topic-item-entity">
             <div class="topic-item-value">
-              <span><[topic.totalSkillCount]></span>
+              <span>{{topic.totalSkillCount}}</span>
             </div>
             <div class="topic-item-type">
               Skills
             </div>
           </div>
           <div class="topic-item-entity topic-item-status">
-            <div class="topic-item-value" ng-if="topic.isPublished">Published</div>
-            <div class="topic-item-value" ng-if="!topic.isPublished">Not Published</div>
+            <div class="topic-item-value" *ngIf="topic.isPublished">Published</div>
+            <div class="topic-item-value" *ngIf="!topic.isPublished">Not Published</div>
           </div>
         </div>
       </div>
     </div>
   </div>
-</md-card>
+</mat-card>
 
 <style>
-  topics-list .list-view-item {
+  .list-view-item {
```

Comment:
Would these styles apply globally? If not, I think I think you might have to change this to "oppia-topics-list .list-view-item" right?

Reply:
No, these are local styles. I don't think we need to add oppia-topics-list (we only do that in ajs not in angular). angular takes care of view encapsulation. So, styles does not effect other directives/components.